src/poetry/installation/executor.py

```diff
@@ -33,6 +33,7 @@
 from poetry.utils.helpers import remove_directory
 from poetry.utils.isolated_build import IsolatedBuildError
 from poetry.utils.isolated_build import IsolatedBuildInstallError
+from poetry.vcs.git import Git
```

Comment:
I assume the `Git` import was local for a reason. It is probably slow and should be avoided in case we do not need it.

Reply:
Would you like me to make the import local to the helper function?

Reply:
> The import in question seems to be already imported in our main branch:

Interesting. That seems to be the import in `direct_origin.py`.

> Would you like me to make the import local to the helper function?

However, even when making this import local, it does not seem to have a significant effect so I suppose we can keep it global. We probably should investigate import times a bit in general before the next release. There seem to be some quick wins.

> time poetry run poetry > /dev/null

You will get better measurements, if you activate the venv manually and than just run `time poetry > /dev/null` so there is less irrelevant code included in the measurement. Further, it might make sense to check the time for different commands, which are expected to be fast, e.g. `poetry -V`.